core/app.go

```diff
@@ -59,14 +64,17 @@ func EmptyApp() *App {
 }
 
 // LoadApp parses the commandline arguments and loads the config
-func LoadApp() (*App, error) {
+func LoadApp() (Runnable, error) {
 
 	var configFlag string
+	var renderFlag string
 	var versionFlag bool
 
 	if !flag.Parsed() {
 		flag.StringVar(&configFlag, "config", "",
 			"JSON config or file:// path to JSON config file.")
+		flag.StringVar(&renderFlag, "render", "",
+		  "- for stdout or file:// path where to save redenred JSON config file.")
```

Comment:
I have no issues with the proposed syntax. The only issue I found, is that the "flag" module does not look like it cleanly supports what you are asking, I am new to Go so I could be wrong here.

My implementation to what you are asking would be the following:
```
...   
flag.StringVar(&renderFlag, "render", "/dev/null", ...)
...

if renderFlag != "/dev/null" {
  // run the rendering code path...
}
```
The issue is that if the user runs `containerpilot -render /dev/null` we will actually end up executing the regular containerpilot application.

If that is still okay I will make the change.